lib/ddtrace/tracer.rb

```diff
@@ -186,23 +187,24 @@ def guess_context_and_parent(child_of)
     # * +start_time+: when the span actually starts (defaults to \now)
     # * +tags+: extra tags which should be added to the span.
     def start_span(name, options = {})
-      start_time = options.fetch(:start_time, Time.now.utc)
+      start_time = options[:start_time] || Time.now
```

Comment:
why this change? is `#fetch` slow?

Reply:
I suggest we add a comment so we don't revert in the future (?)

Reply:
Added comment!